eis_toolkit/cli.py

```diff
@@ -871,6 +871,43 @@ def local_morans_i_cli(
     typer.echo(f"Local Moran's I completed, output vector saved to {output_vector}.")
 
 
+# FEATURE IMPORTANCE
+@app.command()
+def feature_importance_cli(
+    model_file: INPUT_FILE_OPTION,
+    input_rasters: INPUT_FILES_ARGUMENT,
+    target_labels: INPUT_FILE_OPTION,
+    output_file: OUTPUT_FILE_OPTION,
+    feature_names: Optional[List[str]] = None,
```

Comment:
Feature names are not optional per definition of the toolkit function right now. However, since the features are the input rasters given to this CLI function, I would remove this parameter from the CLI function and simply generate the required `feature_names` (so list of input raster names).